rotkehlchen/tests/db/test_history_events.py

```diff
@@ -179,6 +180,33 @@ def test_read_write_events_from_db(database):
                 assert event == expected_event
 
 
+def test_read_write_customized_events_from_db(database):
+    db = DBHistoryEvents(database)
+    history_data = {
```

Comment:
can you also add a evm event and a staking event and filter them? Also add a small docstring explaining what this is doing

Reply:
Updated as suggested.